### PR TITLE
Triples natural firestack decay rate

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Triples natural firestack decay rate, from 0.05 (!) to 0.15

## Why It's Good For The Game

Fire stacks take SO long to decay it's genuinely insane. Like, oh my god. It's so long. They just DON'T. CLEAR. If you're in crit and ignited you're FUCKED. It's suuuuuuuuper annoying becaue it means you always need to treat fire like an emergency, even though it's incredibly common to get a minor amount of ignition from things like cigars, sparks, welding fuel...

Look at this:
image
It took me seven minutes for flame to decay. Seven whole-ass minutes. With a elite modsuit!!

Fire will still overheat you, will still burn your clothes off. This just makes it so that welding fuel fire isn't literally fucking napalm.

## Changelog

:cl:
balance: Triples natural firestack decay rate
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
> If you're in crit and ignited you're FUCKED.  
If you're unconscious and on fire then you will die? I think that's the game working as intended...  
If fire goes out while you are in crit without you actively rolling around that is bad and shouldn't happen.  
If it takes 7 minutes for fire to go out by itself _when you are taking no active action to extinguish it_ I have no problem with that. I wouldn't care if it _never_ went out.

**Comment by MrMelbert** [comment]
I don't agree to the justification in three parts

> If you're in crit and ignited you're FUCKED

You should be

> It's suuuuuuuuper annoying becaue it means you always need to treat fire like an emergency

You should be

> This just makes it so that welding fuel fire isn't literally fucking napalm.

It isn't, though I agree that it could have more nuance afforded

**Comment by carlarctg** [comment]
> It isn't, though I agree that it could have more nuance afforded

perhaps this is the path forward - fire seems to always be intensity max, with the duration being what changes. it should likely have both intensity and duration

**Comment by Namwty** [comment]
> It's suuuuuuuuper annoying becaue it means you always need to treat fire like an emergency

It should be? Even then, at best... fire right now is "Oh shit, my boots/jumpsuit/suit are burning" type of threat. No one gives a shit about burning damages, especially if you got lenturi and what else, cause lol, that shit straight up overheals is.

Right now, burning only causes one issue, and that's losing clothes. No one actually cares about burning ( I do belive there was PR about burning doing actually dmg, but it didn't pass, I think ( think it was goofball good pr ) )

Minor ignitions from my perspective, never actually cause issues, heck you can remove those items before they cause damages ( like miners using black gloves ).


TL;DR

Fire is useless, majority of players only care cause it burns clothes, which makes you lose items.... that's it. Even most basic meds, actually overheal the fucking temperature dmg, heck ya can even use cola and other meds to cool yourself down.

Like literally, being on fire is just a threat of you being naked... that's it.

And honestly, with PR passing or not, it's not gonna make fire anything more than "Lol my jumsuit burned and ID fell of me", unless we talking about absolute green pins.

**Comment by MrMelbert** [comment]
> perhaps this is the path forward - fire seems to always be intensity max, with the duration being what changes. it should likely have both intensity and duration

I would really like if we made fire per-bodypart, so we can have a lighter just ignite your arm/hand - which would do less damage overall damage, not burn all your clothes, etc

**Comment by Iamgoofball** [comment]
Have you heard of a fire extinguisher?

**Comment by Singul0** [comment]
the nu-rad-ification of fire.....

**Comment by moocowswag** [comment]
It should decay faster in space or in the snow outside in icebox.

**Comment by Loafin34** [comment]
where's the [IDED]

**Comment by TheVekter** [comment]
I don't normally complain about hugbox changes, so I hope you understand how serious I am when I say this is a hugbox change.

**Comment by Dorsisdwarf** [comment]
I'll be honest I thought fire on a body just doesn't ever go out unless actively extinguished, those things just burn forever in my experience. 

I dont know if just slashing it by 2/3rds really solves the underlying problems of fire being either absolutely ignorable or "Sorry you got put on fire you immediately lost all your gear and got infected burns and there was nothing you could do" though so meh

**Comment by ZephyrTFA** [comment]
I do not see this ever being merged by anyone; nor you being able to justify this in a way that isn't just ided
